pages/page.js

```diff
@@ -2,5 +2,17 @@ class HomePage {
     async open(path) {
         await browser.url(path);
     }
+    async createNewPaste(testData) {
+        await this.newPasteField.setValue(testData.text);
+        await this.adCloseButton.click(); // close Advertisment Bunner          
+        if (testData.syntax) {
+            await this.checkBoxSyntaxHighlight.click();        
+            await this.dropDownMenuHighlght.click();
+            await this.valueBashHighlghtMenu.click(); 
+        };
+        await this.dropDownMenuExpiration.click();
+        await this.value10MinExpirationMenu.click();               
+        await this.titleField.setValue(testData.title);
+    }
```

Comment:
This method should be implemented in NewPastePage class. Class Page is default abstract page.

Reply:
Thank you! I added new commit (36a032) where transferred createNewPaste method from the Page class to the NewPastePage class.